i18n/en-US.properties

```diff
@@ -58,6 +58,16 @@ be.choose = Choose
 be.close = Close
 # Message to the user to collapse the Transcript entries
 be.collapse = Collapse
+# Text to show on menu item to delete the annotation
+be.contentSidebar.activityFeed.annotationActivity.annotationActivityDeleteMenuItem = Delete
+# Confirmation prompt text to delete the annotation
+be.contentSidebar.activityFeed.annotationActivity.annotationActivityDeletePrompt = Are you sure you want to permanently delete this comment?
```

Comment:
Should this be specific to comments?

Reply:
I need to follow up on the exact wording, but that is what it shows in the design.